generators/test/templates/template_test.go

```diff
@@ -0,0 +1,57 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/gruntwork-io/terratest/modules/test-structure"
+)
+
+func TestTerraformSshExample(t *testing.T) {
+	t.Parallel()
+
+	exampleFolder := "./fixture"
+
+	// At the end of the test, run `terraform destroy` to clean up any resources that were created
```

Comment:
remove "run `terraform` destroy" here, no need to explain underlying implementation, just its functionality.